native/cocos/editor-support/spine/Vector.h

```diff
@@ -158,6 +158,12 @@ class SP_API Vector : public SpineObject {
         return -1;
     }
 
+    inline const T &operator[](size_t inIndex) const {
+        assert(inIndex < _size);
```

Comment:
Why add this function?

Reply:
Spine does not provide const Vector operator[] in C++ to go to TS API, which needs to be converted to std::vector and needs to be accessed. I added this API after communicating with Jianhua.
<img width="759" alt="Screenshot 2023-08-22 at 16 26 58" src="https://github.com/cocos/cocos-engine/assets/2398658/19f3892c-8ea6-4e9a-851c-88fb50eeb9c7">

Reply:
change copy code Vector<String> attachmentNames = obj.getAttachmentNames();